parse_sdf_utils.py

```diff
@@ -550,8 +553,8 @@ def get_dataset_from_record(fnames,
     ValueError: if <fname>.info does not exist for any files in fnames.
   """
   if mode == tf.estimator.ModeKeys.TRAIN and len(fnames) > 1:
-    raise ValueError('Loading multiple files are not supported by this function'
```

Comment:
Why remove this check?

Reply:
I removed the check since the repository can now handle uploading multiple files. I updated the warning to remind people to set a shuffle buffer large enough to shuffle over all input files.